charts/tidb-cluster/templates/tidb-cluster.yaml

```diff
@@ -16,6 +16,9 @@ spec:
   pd:
     replicas: {{ .Values.pd.replicas }}
     image: {{ .Values.pd.image }}
+  {{- if .Values.pd.imagePullPolicy }} 
+    imagePullPolicy: {{ .Values.pd.imagePullPolicy }}
```

Comment:
use `imagePullPolicy: {{ .Values.pd.imagePullPolicy | default "IfNotPresent" }}` to set a default value. This can avoid errors when users use an old values.yaml file which doesn't contain this value.

Reply:
modified